src/trix/models/composition.coffee

```diff
@@ -303,6 +323,9 @@ class Trix.Composition extends Trix.BasicObject
       commonAttributes = @document.getCommonAttributesAtRange(selectedRange)
       unless objectsAreEqual(commonAttributes, @currentAttributes)
         @currentAttributes = commonAttributes
+        for blockAttribute in Object.keys(Trix.config.blockAttributes)
```

Comment:
Seems like we should be passing block *and* text attributes through the `@canSetCurrentAttribute` check, and `@canSetCurrentAttribute` should be updated to handle both. Right now it would disable all text attributes based on your changes to it.